app/scripts/filters/resources.js

```diff
@@ -1093,6 +1105,13 @@ angular.module('openshiftConsole')
       return _.find(_.get(apiObject, 'status.conditions'), {type: type});
     };
   })
+  // returns true/false based on the presence of a Ready status conditions
+  // statusConditionReady(obj)
+  .filter('statusConditionReady', function(statusConditionFilter) {
```

Comment:
Nit: I might give this a name to make it clear it's specific to service catalog items. Maybe `isInstanceOrBindingReady` since it doesn't apply to other types (a bit long I know)